lib/usd/ui/CMakeLists.txt

```diff
@@ -0,0 +1,131 @@
+#
+# =======================================================================
+# Copyright 2019 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =======================================================================
+#
+
+cmake_minimum_required(VERSION 3.1.1)
+project(mayaUsdUI)
+
+if (POLICY CMP0074)
+    cmake_policy(SET CMP0074 OLD)
+endif()
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_PREFIX_PATH "${QT_LOCATION}")
```

Comment:
Your right sorry I do not need "CMAKE_INCLUDE_CURRENT_DIR". That is only needed for older cmake versions.
But without CMAKE_PREFIX_PATH the find_package(Qt5) will fail because it does not know where Qt is located. I followed what is suggested here: https://doc.qt.io/qt-5/cmake-get-started.html#build-a-gui-executable
Although I set it directly here rather than in an env var.